### ISSUE TITLE
The Board used in XY-Plotter V2.0 is based on Arduino UNO, not Leonardo

### ISSUE THREAD

**Issue by Prunebutt** [Makeblock-official/XY-Plotter-2.0#15]
According to the manufacturer homepage and the Microcontroller used on the board, which says “ATMEGA328P” (Arduino UNO), the XY-Plotter V2.0 board is based on the Arduino UNO.

**Comment by guillermoperezdelpulgar** [comment]
I agree! Otherwise it does nos update correctly. At least in my case.

**Comment by shimpe** [comment]
https://github.com/shimpe/XY-Plotter-2.0/ contains updated firmware and electrical connection instructions to make the UNO based plotters work

If you got stuck, maybe worth an attempt :)
